src/NuGet.Clients/NuGet.PackageManagement.UI/PackageItemLoader.cs

```diff
@@ -270,7 +270,8 @@ public IEnumerable<PackageItemListViewModel> GetCurrent()
                         Summary = metadata.Summary,
                         Versions = AsyncLazy.New(() => metadata.GetVersionsAsync()),
                         AllowedVersions = allowedVersions,
-                        PrefixReserved = metadata.PrefixReserved && !IsMultiSource
+                        PrefixReserved = metadata.PrefixReserved && !IsMultiSource,
+                        IsPackageDeprecated = metadata.DeprecationMetadata != null,
```

Comment:
Should we be displaying deprecation metadata if this is a multi source? 
@anangaur Specifically the all view. It'd be an inconsistent experience. 

We already do the same thing with prefix reservation.

Reply:
By extent: the same question will surface when we want to display vulnerability data...
IMHO, as soon as one source states that a package is vulnerable, it should be displayed to the user. I think the same is true for deprecation info. If the 'All sources' view is the default (didn't check), then these features will not be very discoverable.

Reply:
We persist the source selected in the vs settings, so it is saved on the machine. 
I don't think we default to all on a new machine though, but I'm not confident about that :)

Reply:
The *All* view does indicate where it's getting it's information from though... so I think it's fine to just show it?

You can try this branch of the client and look for this particular package and enable both DEV, INT and PROD sources. Select *All* view, and toggle between package versions. Imho, the experience is consistent?

![image](https://user-images.githubusercontent.com/880728/60886470-63fc2e00-a252-11e9-9263-2dd9b3bd6802.png)

Reply:
What if there are more packages on one feed? And that one does not provide deprecation information. 

Then you'd get things like: 

1.0.0 - deprecated. 
1.0.1 (from different feed, not deprecated)
2.0.0 - deprecated. 

Even if you try to exclude deprecated it will be a funny experience.

Reply:
That's exactly what you could simulate using the above feeds and package ID ;)

It would expose versions on several feeds, some deprecated, some not.
Still, the info shown is correct as the used source is also displayed accordingly.
At least the UI is consistent with the data on each source :)

Reply:
Oh I'm not suggesting it will a bug in how it is displayed. That will be consistent. We use the result from 1 source for that id/version combo.

I'm just arguing the inconsistency can be confusing.